tests/Functional/ContentRenderTestBase.php

```diff
@@ -321,4 +322,62 @@ protected function assertLinkIcon(NodeElement $element, string $title, string $h
     $this->assertContains($icon_type, $icon[0]->getAttribute('xlink:href'));
   }
 
+  /**
+   * Creates Publication Document reference entity.
+   *
+   * @param int $status
+   *   Entity status.
+   *
+   * @return \Drupal\oe_content_sub_entity_document_reference\Entity\DocumentReference
+   *   Document reference publication entity.
+   */
+  protected function createPublicationDocumentReferenceEntity(int $status): DocumentReference {
+    $document = $this->createMediaDocument('document');
+    /** @var \Drupal\node\Entity\Node $publication */
+    $publication = $this->getStorage('node')->create([
+      'type' => 'oe_publication',
+      'title' => 'Publication node',
```

Comment:
We need to be able to set "assertable values" from the outside caller, so that, for example, we can create two publication nodes, "Publication 1" and "Publication 2" and use then in the test in different context. Let's make sure that at least the title can be set by the caller.